RadixWallet/Profile/PerNetwork/Profile+Network.swift

```diff
@@ -15,7 +15,7 @@ extension Profile {
 		/// have been added and dApps connected.
 		public let networkID: NetworkID
 
-		public typealias Accounts = NonEmpty<IdentifiedArrayOf<Account>>
+		public typealias Accounts = IdentifiedArrayOf<Account>
```

Comment:
quite a change, now we do allow networks to have zero accounts?

Reply:
Yes yes in fact we do. It is not most users would be doing... but it allows for `New User -> Recover Wallet Without Profile -> Import Mnemonic -> Account Recovery Scan -> No accounts found -> Continue`, voila, in Home, without any accounts. which was something we agreed to not prevent.

Reply:
ok, I thought that maybe in this case, with no accounts found, we would prompt users to create an account.